### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by lambdalisue**
Thanks for the great plugin. I've applied some refactoring so that users can

- Use mods like `:verbose Rapidfire hoge` then `echo &verbose` shows 1
- Use `g:rapidfire#commands` instead of calling functions
- Use `RapidfirePre/RapidfirePost` autocmd to do something

But it's up to you 👍

### CODE REVIEW

**Comment by lambdalisue** [comment]
With this refactoring, I could make a plugin of rapidfire like
https://github.com/lambdalisue/rapidfire-persistent.vim

File: plugin/rapidfire.vim [hunk 1]
```diff
@@ -3,6 +3,4 @@ if exists('g:rapidfire_loaded')
 endif
 let g:rapidfire_loaded = 1
 
-command! -nargs=1
-      \ Rapidfire
-      \ call rapidfire#command#rapidfire#call(<q-mods>, <f-args>)
+command! -nargs=1 Rapidfire call rapidfire#call(<q-mods>, <f-args>)
```
Comment (c000) [inline plugin/rapidfire.vim:6]: Do you have any rules about vim script line breaks?

**Review by c000 — APPROVED** [review]
Thanks for refactoring!

I have a small question.
If you have a moment, please answer them.

File: plugin/rapidfire.vim (hunk 1 above)
Comment (lambdalisue) [reply plugin/rapidfire.vim:6]: No. I don't have any rules or favor or whatever. But some people do for example, some people don't like the default indent level for line breaks so they changed.

**Comment by lambdalisue** [comment]
Thanks! 🎉

**Pull request merged**